src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -183,6 +184,10 @@ public static function createConnection($dsn, array $options = [])
                 $host = $hosts[0]['host'] ?? $hosts[0]['path'];
                 $port = $hosts[0]['port'] ?? null;
 
+                if ($params['tls']) {
+                    $host = 'tls://'.$host;
```

Comment:
sure that this works with `predis` and `phpredis`

Reply:
This is only going to be applied for `phpredis`. I believe this should be done outside of the `\Redis` condition.

Reply:
@misaert Can you change this so it's applied to `\Redis`, `\RedisArray`, `\RedisCluster` and `\Predis\ClientInterface`?

Reply:
Yes, I will do it tomorrow.

Reply:
I changed the code but I do not test yet.